backend/src/govsg/services/govsg-callback.service.ts

```diff
@@ -406,10 +421,18 @@ const parseUserMessageWebhook = async (
   body: UserMessageWebhook,
   clientId: WhatsAppApiClient
 ): Promise<void> => {
-  const { wa_id: whatsappId } = body.contacts[0]
-  const { id: messageId, type } = body.messages[0]
+  await Promise.all(
```

Comment:
I'm wondering if the ordering of the items mean anything? Can we process them concurrently or must they be processed sequentially?

Reply:
same as above, will always be array of length 1

Reply:
Replied in the thread below!